### DISCUSSION TITLE
The making of Chronos :)

### DISCUSSION THREAD

**Discussion by Nitya-Pasrija** [aceta-minophen/Chronos#8] [Developer's Joint]
![image](https://user-images.githubusercontent.com/97171261/158004300-e887932a-78e1-4f3d-9ea1-e3eb49cc2e30.png)
![image](https://user-images.githubusercontent.com/97171261/158004312-3dad6ac5-5828-4849-805b-98473d53229c.png)

Do we add a play thing feature? How does the picture look? oh oh and does the wake words thing feel alright?

**Comment by aceta-minophen** [comment]
> ![image](https://user-images.githubusercontent.com/97171261/158004300-e887932a-78e1-4f3d-9ea1-e3eb49cc2e30.png) ![image](https://user-images.githubusercontent.com/97171261/158004312-3dad6ac5-5828-4849-805b-98473d53229c.png)
> 
> Do we add a play thing feature? How does the picture look? oh oh and does the wake words thing feel alright?

ig we can chenge the wakeword !tell to !calc?

**Reply by Nitya-Pasrija** [reply]
sure that can be done :) I was just used !tell it as we needed the bot workflow structure XD
